cms/templates/widgets/header.html

```diff
@@ -1,18 +1,17 @@
-<%page expression_filter="h"/>
+<%page expression_filter="h" args="online_help_token"/>
 <%namespace name='static' file='../static_content.html'/>
 <%!
   from django.conf import settings
   from django.core.urlresolvers import reverse
   from django.utils.translation import ugettext as _
   from contentstore.context_processors import doc_url
 %>
-<%page args="online_help_token"/>
 <div class="wrapper-header wrapper" id="view-top">
   <header class="primary" role="banner">
 
     <div class="wrapper wrapper-l">
       <h1 class="branding"><a href="/">
-        <img src="${static.url("images/studio-logo.png")}" alt="${settings.STUDIO_NAME}" />
+        <img src="${static.url('images/studio-logo.png')}" alt="${settings.STUDIO_NAME}" />
```

Comment:
My fixes didn't include these quote changes, so up to you whether you want to keep this.